src/rollback_to_stable/rts_btree.c

```diff
@@ -165,7 +174,13 @@ __rts_btree_abort_insert_list(WT_SESSION_IMPL *session, WT_PAGE *page, WT_INSERT
                  * the impacts of a long term correction in WT-10017. Once completed this change can
                  * be reverted.
                  */
+                __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_3,
+                  WT_RTS_VERB_TAG_HS_ABORT_CHECK "insert list with durable_ts=%" PRIu64
+                                                 " had no stable updates. Check the history store "
```

Comment:
Not sure why `s_all` doesn't seem to have picked these up, but the convention is that we only indent by 2 in these situations. There's a couple of places like this, can you take a scroll through the diff and check please?